tools/shared/src/main/scala/org/scalajs/core/tools/javascript/IncClassEmitter.scala

```diff
@@ -0,0 +1,68 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js tools             **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2016, LAMP/EPFL        **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+
+package org.scalajs.core.tools.javascript
+
+import scala.collection.mutable
+
+import org.scalajs.core.tools.sem._
+
+import org.scalajs.core.tools.linker.LinkingUnit
+
+/** Tracks incrementality breaking changes made during class emitting.
+ */
+final class IncClassEmitter (semantics: Semantics, outputMode: OutputMode) {
+  private var _classEmitter: ScalaJSClassEmitter = _
```

Comment:
Actually, I think this is never used as a field. It can be a local `val` in `beginRun`.